lib/python/ray/worker.py

```
@@ -652,13 +652,30 @@ def get_address_info_from_redis_helper(redis_address, node_ip_address):
       elif info[b"client_type"].decode("ascii") == "photon":
         local_schedulers.append(info)
   # Make sure that we got at one plasma manager and local scheduler.
-  assert len(plasma_managers) == 1
-  assert len(local_schedulers) == 1
+  assert len(plasma_managers) >= 1
+  assert len(local_schedulers) >= 1
+  # Build the address information.
+  object_store_addresses = []
+  for manager in plasma_managers:
+    address = manager[b"address"].decode("ascii")
+    try:
+      port = int(address.split(":")[1])
+    except:
```

We should probably throw the exception.